dm/ctl/master/operate_leader.go

```diff
@@ -28,7 +28,7 @@ import (
 func NewOperateLeaderCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "operate-leader <operate-type>",
-		Short: "operate-type can be 'evict' or 'cancel-evict', used to evict leader or cancel evict",
+		Short: "evict/cancel-evict the leader.",
```

Comment:
how about `evict`/`cancel-evict` to indicate it's a part of command (and ditto for below commands)

Reply:
address in 1caa648